src/js/stores/UserSettingsStore.ts

```diff
@@ -36,6 +37,27 @@ class UserSettingsStore extends EventEmitter {
   get storeID() {
     return "userSettings";
   }
+
+  get JSONEditorExpandedSetting(): boolean {
+    const settings = (this.getKey(JSON_EDITOR_SETTING) || {}) as {
```

Comment:
maybe we can do without a type-cast?

that would most likely also make sure that our code can not possibly break here (which it currently could, if `this.getKey(JSON_EDITOR_SETTING)` returned something truthy that's not an object.

Reply:
fixed